cortx-motr.spec.in

```diff
@@ -313,6 +313,7 @@ fi
 %config %attr(0664, motr, motr) %{_sysconfdir}/sysconfig/*
 %config %attr(0755, root, root) %{_sysconfdir}/cron.hourly/m0trace*
 %config %attr(0755, root, root) %{_sysconfdir}/cron.hourly/m0addb*
+%config %attr(0755, root, root) %{_sysconfdir}/cron.hourly/motr*
```

Comment:
instead of motr* use motr_mini_prov* to be more precise.

Reply:
Resolved 7230b13